kolibri/core/assets/src/exams/utils.js

```diff
@@ -15,7 +14,7 @@ import { ExamResource, ContentNodeResource } from 'kolibri.resources';
  * @returns {array} - pseudo-randomized list of question objects compatible with v1 like:
  *    { exercise_id, question_id }
  */
-function convertExamQuestionSourcesV0V2(questionSources, seed, questionIds) {
+function convertExamQuestionSourcesV0V1(questionSources, seed, questionIds) {
```

Comment:
This function name has been changed, but the output of this function hasn't changed at all, so what justifies the name change here?